src/api/models/Business.ts

```diff
@@ -62,6 +55,11 @@ Business.init(
         name: {
             type: DataTypes.STRING,
             allowNull: false
+        },
+        address: {
+            type: DataTypes.STRING,
+            allowNull: false,
+            defaultValue: 'Nowhere'
```

Comment:
Same concern as above for the default address value

Reply:
+1